lib/workers/repository/__fixtures__/master-issue_with_8_PR.txt

```diff
@@ -20,6 +21,7 @@ These updates are currently rate limited. Click on a checkbox below to force the
 
```

Comment:
I think it looks better if we use `rate-limited` with the `-` in between the words.
To be consistent we should change:

- the `h2` heading to `## Rate-limited`
- the sentence to `These updates are currently rate-limited.`

Reply:
agreed